devtools-ci-index.yaml

```diff
@@ -349,6 +349,25 @@
             oc scale --replicas=0 dc almighty-performance
             oc logs dc/almighty-performance --follow
 
+- job:
+    name: 'devtools-test-perfcake-performance-core-db'
+    defaults: global
+    node: devtools
+    properties:
+        - github:
+            url: https://github.com/ldimaggi/perfcake
+    scm:
+        - git:
+            url: https://github.com/ldimaggi/perfcake.git
+            shallow_clone: true
+            branches:
+                - master
+    triggers:
+        - timed: '15 00 * * *'
```

Comment:
I would change this to be sometime during the work day, maybe 30 min after the relevant person gets into work ?